docs/Best-Practices/Handling-Rate-Limit-Errors.md

```diff
@@ -164,41 +102,23 @@ In this scenario, let's assume your application has six access tokens (Token A,
         `employment` | 0/20
         `payment` | 4/12
         `pay-statement` | 0/12
-1. Token C, D, and E repeats the same process as Token B making 5 requests to the [/employer/company](https://developer.tryfinch.com/docs/reference/33162be1eed72-company) endpoint, 3 requests to the [/employer/directory](https://developer.tryfinch.com/docs/reference/12419c085fc0e-directory) endpoint, and 2 requests to the [/employer/payment](https://developer.tryfinch.com/docs/reference/b811fdc2542ca-payment) endpoint all within the same minute. Similarly, every 5th request to the [/employer/company](https://developer.tryfinch.com/docs/reference/33162be1eed72-company) endpoint for each token fails and returns a 429 rate limit error.
-    - Access Token C, D, and E individual rate limits
-        Bucket | Capacity
-        -------|-------------
-        `company` | 4/4 (FULL)
-        `directory` | 3/4
-        `individual` | 0/4
-        `employment` | 0/4
-        `payment` | 2/2 (FULL)
-        `pay-statement` | 0/2
+1. Token C, D, and E repeats the same process as Token B making 5 requests to the [/employer/company](https://developer.tryfinch.com/docs/reference/33162be1eed72-company) endpoint, 3 requests to the [/employer/directory](https://developer.tryfinch.com/docs/reference/12419c085fc0e-directory) endpoint, and 2 requests to the [/employer/payment](https://developer.tryfinch.com/docs/reference/b811fdc2542ca-payment) endpoint all within the same minute. The requests will succeed until the 20th request to the  [/employer/company](https://developer.tryfinch.com/docs/reference/33162be1eed72-company) endpoint, after which point each token fails and returns a 429 rate limit error.
     - Application-level rate limits
         Bucket | Capacity
         -------|-------------
         `company` | 20/20 (FULL)
         `directory` | 15/20
         `individual` | 0/20
         `employment` | 0/20
-        `payment` | 12/12 (FULL)
+        `payment` | 10/12
```

Comment:
Turns out the docs also did some suspiciously bad addition for the payment endpoint in this scenario 🙄 

https://developer.tryfinch.com/docs/reference/b288acfbb6816-handling-rate-limit-errors

Reply:
Nice catch -- love the attn to detail